### DISCUSSION TITLE
Does the MO federal tax deduction apply to negative federal tax?

### DISCUSSION THREAD

**Discussion by MaxGhenis** [PolicyEngine/policyengine-us#1375] [Q&A] [answered]
Missouri has a federal income tax deduction: regardless of itemization, filers deduct a percentage of their federal tax liability which phases out with income.

Aside from the circular logic with respect to the SALT deduction (#1050), it's unclear how MO defines federal income tax for this purpose. The [form](https://dor.mo.gov/forms/MO-1040%20Instructions_2021.pdf#page=22) only says this (1040 line 11), and the instructions don't provide any more detail:
>Total tax from federal return. Do not enter federal income tax withheld.

The [law](https://revisor.mo.gov/main/OneSection.aspx?section=143.171&bid=49937&hl=federal+income+tax+deduction%u2044) says:
>an individual taxpayer shall be allowed a deduction equal to a percentage of his or her federal income tax liability under Chapter 1 of the Internal Revenue Code for the same taxable year for which the Missouri return is being filed, not to exceed five thousand dollars on a single taxpayer's return or ten thousand dollars on a combined return, after reduction for all credits thereon, except the credit for payments of federal estimated tax, the credit for the overpayment of any federal tax, and the credits allowed by the Internal Revenue Code by 26 U.S.C. Section 31, 26 U.S.C. Section 27, and 26 U.S.C. Section 34.  

Sections 31, 27, and 34 do not refer to refundable credits like the EITC and CTC.

We've currently implemented this as including negative taxes, as that's the most direct reading of the form. However, the law's "Shall be allowed" implies that filers don't have to claim it. I emailed the MO DOR to request clarification.

**Comment by nikhilwoodruff** [comment, marked as answer]
Yeah, the "shall be allowed" making this voluntary was my reading too.

**Reply by MaxGhenis** [reply]
The [instructions](https://dor.mo.gov/forms/MO-1040%20Instructions_2021.pdf#page=7) confirm this:
>If a negative amount is calculated, enter “0”.
